test/e2e/logs_test.go

```diff
@@ -311,4 +311,16 @@ var _ = Describe("Podman logs", func() {
 		logs.WaitWithDefaultTimeout()
 		Expect(logs).To(Not(Exit(0)))
 	})
+
+	It("follow output stopped container", func() {
+		containerName := "logs-f"
+
+		logc := podmanTest.Podman([]string{"run", "--name", containerName, "-d", ALPINE})
```

Comment:
I'm not comfortable with this usage (relying on ALPINE's entrypoint to exit immediately). Could you add an explicit command such as `true`?

Reply:
Change has been made and pushed.